packages/epics/src/treasury/components/assets/deposit-funds.tsx

```diff
@@ -0,0 +1,76 @@
+import Link from 'next/link';
+import { Button, Separator, Input } from '@hypha-platform/ui';
+import { RxCross1 } from 'react-icons/rx';
+import { Text } from '@radix-ui/themes';
+import QRCode from 'react-qr-code';
+import { CopyIcon, CheckIcon, Cross2Icon } from '@radix-ui/react-icons';
+import { copyToClipboard } from '@hypha-platform/ui-utils';
+
+interface DepositFundsProps {
+  closeUrl: string;
+  description?: string;
+  address: string;
+}
+
+export const DepositFunds = ({
+  closeUrl,
+  description,
```

Comment:
Is there a reason we pass in `description` but hard-code the `title`? I'd go with hardcoding the description text until we need it dynamic.